cmd/odf/restore/crds.go

```diff
@@ -1,12 +1,55 @@
 package restore
 
 import (
+	"fmt"
+	"slices"
+	"strings"
+
 	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
 	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
 	pkgrestore "github.com/rook/kubectl-rook-ceph/pkg/restore"
 	"github.com/spf13/cobra"
 )
 
+// groupVersions defines the supported CRD groups and their corresponding API versions.
+var groupVersions = map[string]string{
+	"ocs.openshift.io":       "v1",
+	"ceph.rook.io":           "v1",
+	"csi.ceph.io":            "v1beta1",
```

Comment:
In 4.19 we decided to keep this as v1alpha1, then in 4.20 it will change to `v1`.
```suggestion
	"csi.ceph.io":            "v1alpha1",
```

Reply:
done